src/content/en/updates/2018/10/devtools.md

```diff
@@ -0,0 +1,277 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Highlight DOM nodes from Live expressions, store nodes as global variables, and more.
+
+{# wf_updated_on: 2018-10-10 #}
+{# wf_published_on: 2018-10-10 #}
+{# wf_tags: chrome71,devtools,devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: Highlight DOM nodes from Live expressions, store nodes as global variables, and more. #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 71) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+New features and major changes coming to Chrome DevTools in Chrome 71 include:
+
+* [Hover over a Live Expression to highlight a DOM node](#hover)
+* [Store DOM nodes as global variables](#store)
+* [Emulate page focus](#focus)
+* [Initiator and priority information now in HAR imports and exports](#HAR)
+* [Access the Command Menu from the Main Menu](#command-menu)
+* ["Add to homescreen" now referred to as "Trigger beforeinstallprompt"](#beforeinstallprompt)
+* [Picture-in-Picture breakpoints](#picture-in-picture)
+
+## Hover over a Live Expression to highlight a DOM node {: #hover }
+
+When a [Live Expression][LE] evaluates to a DOM node, hover over the Live Expression result to
+highlight that node in the viewport.
+
+[LE]: /web/updates/2018/08/devtools#watch
+
+<figure>
+  <img src="/web/updates/images/2018/10/hover1.png"
+       alt="Hovering over a Live Expression result to highlight the node in the viewport."/>
+  <figcaption>
+    <b>Figure 1</b>. Hovering over a Live Expression result to highlight the node in the viewport
+  </figcaption>
+</figure>
+
+## Store DOM nodes as global variables {: #store }
+
+To store a DOM node as a global variable, run an expression in the Console that evaluates to
+a node, right-click the result, and then select **Store as global variable**.
+
+<figure>
+  <img src="/web/updates/images/2018/10/store1.png"
+       alt="Store as global variable in the Console."/>
+  <figcaption>
+    <b>Figure 2</b>. Store as global variable in the Console
+  </figcaption>
+</figure>
+
+Or, right-click the node in the **DOM Tree** and select **Store as global variable**.
+
+<figure>
+  <img src="/web/updates/images/2018/10/store2.png"
+       alt="Store as global variable in the DOM Tree."/>
+  <figcaption>
+    <b>Figure 3</b>. Store as global variable in the DOM Tree
+  </figcaption>
+</figure>
+
+## Emulate page focus {: #focus }
```

Comment:
"Ever struggle to debug styles that are different when the page has focus? Suppose you've got a collection of buttons..."